packages/-ember-data/tests/integration/adapter/rest-adapter-test.js

```diff
@@ -313,8 +313,14 @@ module('integration/adapter/rest_adapter - REST Adapter', function (hooks) {
   });
 
   test('updateRecord - hasMany relationships faithfully reflect simultaneous adds and removes', async function (assert) {
-    Post.reopen({ comments: DS.hasMany('comment', { async: false }) });
+    Post = DS.Model.extend({
+      name: DS.attr('string'),
+      comments: DS.hasMany('comment', { async: false }),
+    });
     Comment.reopen({ post: DS.belongsTo('post', { async: false }) });
+
+    this.owner.register('model:post', Post);
```

Comment:
The `hooks.beforeEach` callback above still registers Post as well. Can we remove the registration in that callback and ensure that any other tests using `post` also register it directly?